MapboxNavigation/InstructionPresenter.swift

```diff
@@ -140,6 +156,23 @@ class InstructionPresenter {
         attachment.image = shieldImage
         return NSAttributedString(attachment: attachment)
     }
+    
+    private func exitShield(side: ExitSide = .right, text: String) -> NSAttributedString {
+        let exit = ExitView(pointSize: label!.font.pointSize, side: side, text: text)
+        exit.translatesAutoresizingMaskIntoConstraints = false
+        exit.invalidateIntrinsicContentSize()
+        exit.setNeedsLayout()
+        exit.layoutIfNeeded()
+        let exitAttachment = NSTextAttachment()
+        let exitImage = exit.imageRepresentation
+        exitAttachment.image = exitImage
+        if let label = self.label {
+            let yOrigin = (label.font.capHeight - exitImage.size.height).rounded() / 2
+            exitAttachment.bounds = CGRect(x: 0, y: yOrigin, width: exitImage.size.width, height: exitImage.size.height)
```

Comment:
Any reason we can't reuse shield attachment bounds?

Reply:
Yep, that's the plan. This is interstitial.